### PR TITLE
Fixed window dragging

### PR DESCRIPTION
**Pull request by dapetcu21**
Unfortunately, I didn't find a cleaner solution, so I implemented the dragging manually.

### CODE REVIEW

File: Messenger.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -310,7 +310,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "python > ${BUILD_ROOT}/MainJSDataURL.m << END\nimport base64\nprint \"NSString* kMainJSDataURL = @\\\"data:image/png;base64,\" + base64.b64encode(open(\"website/app/main.js\", \"rb\").read()) + \"\\\";\"\nEND";
+			shellScript = "python > ${BUILD_ROOT}/MainJSDataURL.m << END\nimport base64\nprint \"NSString* kMainJSDataURL = @\\\"data:application/javascript;base64,\" + base64.b64encode(open(\"website/app/main.js\", \"rb\").read()) + \"\\\";\"\nEND";
```
Comment (rsms) [inline Messenger.xcodeproj/project.pbxproj:313]: Oops! Good catch :)

**Comment by rsms** [comment]
Nice. Have you tried building against the 10.10 SDK? (even better: tried it on 10.10)
I don't see why it wouldn't work, but on <=10.10 the title bar view is modified to extend over the header of the app, which is how dragging a larger area works, so potentially there could be a conflict of both views trying to drag the window.

**Comment by dapetcu21** [comment]
I don't seem to have the 10.10 SDK installed. And no, I don't have a 10.10 at hand to test. I suppose it should be fine, though. Worst case scenario, I hide that view conditionally on 10.10.

**Comment by dapetcu21** [comment]
I think that on 10.10, the `mouseDragged` event will be consumed by the `_titleBar` instead of my custom view, since its container has `_window.contentView` as a sibling and is above it.

**Comment by rsms** [comment]
Let's merge it into the OSX10.11 branch for now. Nice work!

**Pull request merged**